### ISSUE TITLE
Economic cost of social distancing

### ISSUE THREAD

**Issue by AdrianListon** [s-andrews/virusbreak#36]
So hard to calculate, but could we use a cost of 200000000 per day? 

This is based on the assumption that social distancing would cost the economy 5% of GDP. Seems fairly conservative.

**Comment by s-andrews** [comment]
Yes, we should be able to add that into the calculation, so that's 200M per day if distancing is turned on in addition to the existing costs.
